src/twisted/names/newsfragments/9786.bugfix

```diff
@@ -0,0 +1 @@
+twistd -n dns --pyzone example-domain.com now works on Python 3.
```

Comment:
Particularly given that this includes `-n`, it should really be `twist dns` (since `twist` never daemonizes, and is the new cool thing anyway).

Reply:
Fixed